public/index.html

```diff
@@ -3,8 +3,116 @@
+          </p>
+          <a href="#">Read More</a>
+          <h2>Rodney Mullen: Pop an ollie and innovate! (TED Talk)</h2>
+          <h4>March 10, 2013</h4>
+          <p>
+            The last thing Rodney Mullen, the godfather of street skating, wanted
+            were competitive victories. In this exuberant talk he shares his love
+            of the open skateboarding community and how the unique environments
+            it plays in drive the creation of new tricks--fostering...
+          </p>
+          <a href="#">Read More</a>
+          <h2>Ray Barbee Talks 'Ban This'</h2>
+          <h4>October 4, 2013</h4>
+          <p>
+            Since filming is in full swing for the Vans Video Adventures With Chris
+            has taken the opportunity to sit some of the team down to look behind
+            the skating of some of their earlier parts. Today we sit down Ray
+            Barbee, one of the smoothest guys to ever step on a skateboard and
+            Chris' childhood idol...
+          <a href="#">Read More</a>
+        </div>
+      </article>
+      <section class="skater_grid">
+        <h2>SKATEBOARDERS</h2>
+        <a class="small_pic" href="#"><img src="img/sk8er-4.png" alt="Photo of Lance">
+          <h1>Lance Mountain</h1></a>
+        <a class="small_pic" href="#"><img src="img/sk8er-3.png" alt="Photo of Rodney">
+          <h1>Roney Mullen</h1></a>
+        <a class="tall_pic" href="#"><img src="img/sk8er-2.png" alt="Photo of Tommy">
+          <h1>Tommy Guerrero</h1></a>
+        <a class="wide_pic" href="#"><img src="img/sk8er-1.png" alt="Photo of Steve">
+          <h1>Steve Caballero</h1></a>
+      </section>
+      <section class="skateboard_grid">
+        <img src="img/sk8-1.png" alt="Image of Skateboard">
+        <img src="img/sk8-2.png" alt="Image of Skateboard">
+        <img src="img/sk8-3.png" alt="Image of Skateboard">
+        <img src="img/sk8-4.png" alt="Image of Skateboard">
+        <img src="img/sk8-5.png" alt="Image of Skateboard">
+        <img src="img/sk8-6.png" alt="Image of Skateboard">
+        <img src="img/sk8-7.png" alt="Image of Skateboard">
+        <img src="img/sk8-8.png" alt="Image of Skateboard">
+        <img src="img/sk8-9.png" alt="Image of Skateboard">
+        <img src="img/sk8-10.png" alt="Image of Skateboard">
+      </section>
+      <section class="bottom_connect">
+        <div class="site_info">
+          <h2>Contact Us for More Information</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+            enim ad minim veniam, quis nostrud exercitation ullamco laboris
+            nisi ut aliquip ex ea commodo consequat.
+            <a href="#">Facebook</a>
+            <a href="#">RSS</a>
+            <a href="#">Twitter</a>
+            <a href="#">YouTube</a>
+          </p>
+          <div class="contact_form">
```

Comment:
Looks like your `contact_form` is inside of the `site_info`. This will make it hard for one to be on the left and the other to be on the right.